src/gtc/common.py

```diff
@@ -572,6 +572,51 @@ def apply(cls, node: Node, *, symtable: Dict[str, Any]) -> List[str]:
     return root_validator(allow_reuse=True, skip_on_failure=True)(_impl)
 
 
+def validate_lvalue_dims() -> RootValidatorType:
```

Comment:
Would be good to have a test for this.

Reply:
Or is this tested in test_gtir.py? Then would be good to move it to test_common.py and decouple the testing from the concrete IR.